.github/workflows/install_deps/action.yml

```diff
@@ -8,6 +8,16 @@ inputs:
     description: The version of Python to use in order to run unit tests
     required: false
     default: '3.10'
+  install_numpy_1:
+    description: Indicate if numpy 1 should be installed or not
+    required: false
+    type: boolean
+    default: false
+  numpy_1_version:
```

Comment:
it looks like nothing is ever passed to this to override the default - why not just use the string where needed? Or does it not work to include strings in the format below?

Reply:
Good point. I removed the `numpy_1_version` variable completely from the scripts since it was unnecessary